packages/web/components/EditProfileModal.tsx

```diff
@@ -545,13 +552,21 @@ export const EditProfileModal: React.FC<ProfileEditorProps> = ({
                 key: 'backgroundImageURL',
                 title: 'Page Background',
                 description:
-                  'An image with an ~1:1 aspect ratio to be the page background. 1MiB maximum size.',
+                  // eslint-disable-next-line prefer-template
+                  'An image with an ~1:1 aspect ratio to be the page background. 1MiB maximum size.' +
+                  (metagamer
+                    ? ''
+                    : '\nOnly available to MetaGame players and patrons.'),
```

Comment:
I don't think restricting features will really serve us at this point in the product's lifespan, but I suppose the bar to become a player/patron is pretty low...

Reply:
I agree.

What I'd personally like to see is an easy to set up easy to theme SourceCred leaderboard with profiles that organizations purchase a subscription for.

In exchange for their money, they don't have to run a Ceramic node or SourceCred.

What I think will be crucial for this is a switch to Credequate & the addition of a second order system that allows people to adjust the algorithmically determined weights.

So far as MetaOS, an operating system really needs flexible and powerful extensibility — like installing programs in a desktop OS, or apps in a mobile OS. Designing and implementing such a system is a huge engineering feat. Maybe someone is going to come up with an elegant and simple answer, but I don't think that what we have is it, nor do I think we have a groundbreaking vision for it.

Simply the fact that all new modules have to be merged into our codebase is a significantly limiting factor both from a security, but also autonomy perspective. To have a real ecosystem, you can't have such a stricture on the publishing lifecycle.

I still say we let MetaOS go for the time being and focus on making it as easy to onboard into SourceCred as filling out a web form with the sources you want to collect and paying a little ether. Add on a program for adjusting the fairness, and I think you have something engaging that people would spend time digging through and playing with.